package.json

```diff
@@ -60,12 +60,13 @@
     "@types/node": "^12.12.24",
     "@types/react": "^16.9.17",
     "@types/react-paginate": "^6.2.1",
-    "autoprefixer": "^9.7.4",
+    "autoprefixer": "^10.2.5",
     "babel-loader": "^8.1.0",
     "bundlesize": "^0.18.0",
-    "jest": "^25.1.0",
+    "jest": "25.1.0",
```

Comment:
I would not care about pinning versions of dev dependencies too much. what's your reasoning for this change?